projects/entry-form/src/styles/modules/components/forms/checkboxs/_general.scss

```diff
@@ -21,10 +21,26 @@
             stroke: map.value-from($theme, 'checkbox.checkmark-color')!important;
         }
 
+        &.mat-checkbox-checked .mat-checkbox-background .mat-checkbox-checkmark-path {}
+
         &:hover {
             .mat-checkbox-frame {
                 border: map.value-from($theme, 'checkbox.border-hover');
             }
         }
     }
+
+    &.mat-checkbox-checked {
+		.mat-checkbox-background {
+			background-color: map.value-from($theme, 'checkbox.selected-background-color');
+
+			.mat-checkbox-checkmark-path {
+				stroke: map.value-from($theme, 'checkbox.selected-checkmark-color')!important;
```

Comment:
What's happening with your stylelint? Formatting seems inconsistent.

Reply:
don't know :/

Reply:
created tickets for that